ui-tests/src/test/java/org/openmrs/reference/AdmitExitInpatientTest.java

```diff
@@ -29,6 +29,7 @@ public void createTestData() throws Exception {
     public void admitExitInpatientTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(testPatient.identifier);
+        findPatientPage.waitForPageToLoad();
```

Comment:
The `search patient indexing criteria` takes a bit long to retrieve the created patient data from the db instance and as a result the patient that is selected from the `pool` does not match the one being searched. Unfortunately in most of the builds the random patient that is selected from the pool has a visit status marked  as `already started (End Visit) ` and the test fails at this [line](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/c0889e948a6b744948cba8d2393445930cdff0e0/ui-tests/src/test/java/org/openmrs/reference/AdmitExitInpatientTest.java#L32)  while looking for the `Start Visit` element which can not be present on the page when the patient visit was already started.  ` waiting the page to load`, ensures that the patient selected from the pool is the test patient that has been created by test.